packages/clerk-js/src/ui/common/CurrentPlanSection.tsx

```diff
@@ -0,0 +1,54 @@
+import type { BillingPlanResource } from '@clerk/types';
+
+import { Box, localizationKeys, Text, useLocalizations } from '../customizables';
+import { ProfileSection } from '../elements';
+import { mqu } from '../styledSystem';
+import { centsToUnit, getRelativeToNowDateKey } from '../utils';
+
+export const CurrentPlanSection = ({ currentPlan }: { currentPlan?: BillingPlanResource | null }) => {
+  const { t } = useLocalizations();
+
+  if (!currentPlan) {
```

Comment:
We can omit this assertion, current plan will always be initiated after the enabling of billing

Reply:
as we discussed offline we are keeping for now